src/cpp/rtps/writer/ReaderProxy.h

```diff
@@ -20,26 +20,23 @@
 
```

Comment:
Take the chance to change the extension of the moved headers to `.hpp`.

Also please change the header guards to follow the pattern `FOLDER_FOLDER__FILENAME_EXTENSION`.

This one would be `RTPS_WRITER__READERPROXY_HPP`

Reply:
Remember to also change them in mocks